Package.swift

```diff
@@ -190,6 +191,15 @@ let package = Package(
     ),
 
     // Cross-import overlays (not supported by Swift Package Manager)
+    .target(
+      name: "_Testing_AppKit",
+      dependencies: [
+        "Testing",
+        "_Testing_CoreGraphics",
+      ],
+      path: "Sources/Overlays/_Testing_AppKit",
+      swiftSettings: .packageSettings
```

Comment:
` + .enableLibraryEvolution()` is encouraged now, and the `_Testing_CoreGraphics` target/overlay has adopted it below